opcUaDevSupApp/drvOpcUa.cpp

```diff
@@ -801,7 +801,9 @@ long OpcWriteValue(int opcUaItemIndex,double val,int verbose)
     return 0;
 }
 /* iocShell: record write func  */
+extern "C" {
```

Comment:
I am afraid that the extern statements in epicsExport.h are ignored by VS compiler when expanding epicsRegisterFunction macro. Therefore they need to be repeated here otherwise we get a linking failure when building test IOC.